### DISCUSSION TITLE
授权无法延期，也无法删除旧授权重新新授权，导致一个账户只能使用24小时

### DISCUSSION THREAD

**Discussion by Giftia** [xisuo67/XHS-Spider#132] [Q&A] [answered]
没有说明这种情况应该如何解决，还是说，原意就是一个账户只能使用一次？

**Comment by xisuo67** [comment, marked as answer]
> 没有说明这种情况应该如何解决，还是说，原意就是一个账户只能使用一次？

如果你有任何疑问可以通过邮件联系我，我的邮箱：admin@xisuo67.website
